### ISSUE TITLE
Could not use keyboard to hit <enter> buttons successfully

### ISSUE THREAD

**Issue by hannyle** [CSCfi/swift-browser-ui#886]
#### Describe the bug

When navigate through SD Connect using **keyboard**, the buttons couldn't work when hitting `Enter` 

#### To Reproduce
Steps to reproduce the behaviour:
1. Go to SD Connect, for example: https://sd-connect-audit-hana.rahtiapp.fi/
2. Use keyboard to navigate to Log in button
3. Try hitting `Enter`
4. See that there is nothing happen after hitting `Enter`

#### Expected behavior

All the buttons in SD Connect should be able to work when hitting `Enter` from keyboard

**Comment by juhtornr** [comment]
This is coming through CSC design system. Might be implemented in Hang's latest branch (if not it will come when we update the package).
